Listenbrainz/UI/Components/UserDetailsView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  UserDetailsView.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 04/12/23.
+//
+
+
+import SwiftUI
+
+struct UserDetailsView: View {
+
+  // MARK: - PROPERTIES
+  @AppStorage("isOnboarding") var isOnboarding: Bool?
```

Comment:
For the app storage names, add these names to the Constants file by creating a struct called Strings or something and using those variables here

Reply:
done